[3468_keyboard] Update some names with Keyboard in name

LornaSIL opened the pull request:
Still in progress while we figure out what to do about legacy.

mcdurdin commented:
Let's leave legacy and deprecated keyboards alone :grin: less work for you I hope

LornaSIL commented:
Ready for review @mcdurdin 
In particular, take a look at `fv_all` and `galaxie_greek_hebrew_mnemonic`. Both of those are packages and I changed "Keyboards" to "Package" for the first one and just removed "Keyboard" in the 2nd one.

LornaSIL commented:
Fixes #3468

mcdurdin commented on legacy/l/laounicodebasic_on_thaikbd/laounicodebasic_on_thaikbd.keyboard_info:
```diff
@@ -1,6 +1,6 @@
 {
   "id": "laounicodebasic_on_thaikbd",
-  "name": "Lao Unicode on Thai Keyboard layout",
+  "name": "Lao using Thai layout",
```
I'm not sure if we should change this one, because 'Lao Unicode' has been a name in popular use to differentiate from other Lao (Unicode) keyboards. Oof.

```suggestion
  "name": "Lao Unicode on Thai Keyboard layout",
```

mcdurdin reviewed (dismissed):
Just one to revert (it's a little bit weird), but apart from that LGTM
